farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.spec.js

```diff
@@ -1332,7 +1332,7 @@ describe('Test the seeding input page', () => {
             })
         })
 
-        it('Minute/Direct Seeding/With comment', () => {
+        it.only('Minute/Direct Seeding/With comment', () => {
```

Comment:
Please remove the only qualifier from any place it is used so that all tests will run.